From a58b76381b96ccf7f0edb180b7ff8984e79763a3 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 13 Mar 2017 16:11:28 +0800
Subject: [PATCH 1646/5242] MLK-14260 video: mipi_dsi_northwest: assign the
 'mode' field of fb_info.

commit  c8a7ad29159ea0a3919e08289ce4a5a4103956cb from
https://source.codeaurora.org/external/imx/linux-imx.git

Assign the 'mode' field of fb_info structure to show correct
content when running "cat /sys/class/graphics/fb0/mode".

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 06a16560683047681c46b680efef7af58c75b6b2)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mipi_dsi_northwest.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/video/fbdev/mxc/mipi_dsi_northwest.c b/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
index 03e4934..0e72a69 100644
--- a/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
+++ b/drivers/video/fbdev/mxc/mipi_dsi_northwest.c
@@ -146,6 +146,8 @@ static int mipi_dsi_lcd_init(struct mipi_dsi_info *mipi_dsi,
 		return err;
 	}
 
+	setting->fbi->mode = mipi_dsi->mode;
+
 	return 0;
 }
 
-- 
1.7.9.5

